client/blocks/conversation-caterpillar/index.jsx

```diff
@@ -30,7 +30,7 @@ class ConversationCaterpillarComponent extends React.Component {
 		// Only display authors with a gravatar, and only display each author once
 		const uniqueAuthors = uniqBy( map( comments, 'author' ), 'ID' );
 		const displayedAuthors = takeRight(
-			filter( uniqueAuthors, 'avatar_URL' ),
+			filter( uniqueAuthors, 'has_avatar' ),
```

Comment:
I started with both, pondered, and then went back to `has_avatar` only. We should never have an avatar URL with `has_avatar` = `false` handed back from the API, should we?